core/editor/config-providers/editor-v1-config-provider.php

```diff
@@ -21,7 +21,10 @@ public function get_script_configs() {
 	}
 
 	public function get_script_handles_to_enqueue() {
-		return [ 'elementor-editor-loader-v1' ];
+		return [
+			'elementor-editor-loader-v1',
+			'elementor-responsive-bar',
```

Comment:
Maybe it will be better if the responsive bar will be one of the deps of the editor, and then you could make `window.addEventListener('elementor/init')` and then use `elementor` global

Reply:
Lets talk about it

Reply:
See above